### ISSUE TITLE
Support umode +B for bots

### ISSUE THREAD

**Issue by Grinnz** [solanum-ircd/solanum#210]
unreal and related ircds support +B as an opt-in usermode. This is highly useful for bots that wish to automatically ignore other bots. Please consider supporting this, or an equivalent mechanism. Thank you.

**Comment by jesopo** [comment]
it feels like bot modes are just to make up for the fact that bots should output on `NOTICE` but don't

**Comment by Grinnz** [comment]
Yes, but this is not possible to fix, since clients do not treat notices acceptably for channel bots to use.

**Comment by xfnw** [comment]
> [...] clients do not treat notices acceptably [,,,]

is this still a real problem? i've used quite a few clients (catgirl, weechat, irssi, hexchat, kirc, thelounge, kiwi, etc) and never had any that do anything unacceptable/weird on NOTICE

**Comment by Grinnz** [comment]
Not unacceptable necessarily, but inappropriate for receiving a reply from a bot in a channel. Such as popups, separate windows, etc.

**Comment by Mikaela** [comment]
Spec: https://ircv3.net/specs/extensions/bot-mode
